src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
testu asta sigur nu mai trece :) daca dai `npm test` in terminal o sa vezi .. incearca sa l imbunatatesti daca poti 😅

Reply:
am cautat pe net si am facut asta npm install --save-dev @testing-library/react

am sa pun asta intr-un commit

Reply:
nu dau solve, pentru ca aparent nu s-a rezolvat :D